### ISSUE TITLE
'Database connection failure' when trying to install Odoo on Easypanel version 1.31.0 and 1.31.1.

### ISSUE THREAD

**Issue by musabatas** [easypanel-io/community#92]
Getting the below error after deploying Odoo16 and Odoo17:

`Database connection failure: could not translate host name "{project_name}_odoo-db" to address: Name or service not known`

**Comment by deiucanta** [comment]
What is your env content? Feel free to redact the passwords

**Comment by musabatas** [comment]
`POSTGRES_DB=postgres
POSTGRES_PASSWORD=***********
POSTGRES_USER=odoo
PGDATA=/var/lib/postgresql/data/pgdata`

Creating postgres13 service with same env content is working, but the deployment from odoo template is not working.

There is an old installation of odoo is also not working after update.

**Comment by deiucanta** [comment]
This seems to be part of the env. I want to see this part `"{project_name}_odoo-db"`

**Comment by musabatas** [comment]
`HOST=$(PROJECT_NAME)_odoo-db
USER=odoo
PASSWORD=**********
PORT=5432`

Here is the odoo container's env. 

But the issue is about database container. It shows successfully deployed in Deployment tab, but container not working. You can check it by deploying Odoo template.

![IMG_20240106_173209](https://github.com/easypanel-io/community/assets/10494850/8efe1477-d563-42b2-a953-b040aa8fd1c9)

**Comment by deiucanta** [comment]
Found the issue and fixed it in v1.31.3. Please upgrade and create a new instance

```
docker image pull easypanel/easypanel && docker service update easypanel --force
```
